Documentation/gettingstarted/k8s-install-helm.rst

```diff
@@ -0,0 +1,242 @@
+          AZURE_CLIENT_ID="$(jq -r .appId < azure-sp.json)"
+          AZURE_CLIENT_SECRET="$(jq -r .password < azure-sp.json)"
+          AZURE_TENANT_ID="$(jq -r .tenant < azure-sp.json)"
+          AZURE_NODE_RESOURCE_GROUP="$(az aks show --resource-group $RESOURCE_GROUP_NAME --name $CLUSTER_NAME | jq -r .nodeResourceGroup)"
+
+       .. note::
+
+          ``AZURE_NODE_RESOURCE_GROUP`` must be set to the resource group of
+          the node pool, *not* the resource group of the AKS cluster.
+
+       **Install Cilium:**
+
+       Deploy Cilium release via Helm:
+
+       .. parsed-literal::
+
+          helm install cilium |CHART_RELEASE| \\
+            --namespace kube-system \\
+            --set azure.enabled=true \\
+            --set azure.resourceGroup=$AZURE_NODE_RESOURCE_GROUP \\
+            --set azure.subscriptionID=$AZURE_SUBSCRIPTION_ID \\
+            --set azure.tenantID=$AZURE_TENANT_ID \\
+            --set azure.clientID=$AZURE_CLIENT_ID \\
+            --set azure.clientSecret=$AZURE_CLIENT_SECRET \\
+            --set tunnel=disabled \\
+            --set ipam.mode=azure \\
+            --set enableIPv4Masquerade=false \\
+            --set nodeinit.enabled=true
+
+    .. group-tab:: AWS/EKS
+
+       .. include:: requirements-eks.rst
+
+       **Delete VPC CNI (``aws-node`` DaemonSet)**
+
+       Cilium will manage ENIs instead of VPC CNI, so the ``aws-node``
+       DaemonSet has to be deleted to prevent conflict behavior.
+
+       .. code-block:: shell-session
+
+          kubectl -n kube-system delete daemonset aws-node
+
+       **Intall Cilium:**
+
+       Deploy Cilium release via Helm:
+
+       .. parsed-literal::
+
+          helm install cilium |CHART_RELEASE| \\
+            --namespace kube-system \\
+            --set eni.enabled=true \\
+            --set ipam.mode=eni \\
+            --set egressMasqueradeInterfaces=eth0 \\
+            --set tunnel=disabled \\
+            --set nodeinit.enabled=true
+
+       .. note::
+
+          This helm command sets ``eni.enabled=true`` and ``tunnel=disabled``,
+          meaning that Cilium will allocate a fully-routable AWS ENI IP address
+          for each pod, similar to the behavior of the `Amazon VPC CNI plugin
+          <https://docs.aws.amazon.com/eks/latest/userguide/pod-networking.html>`_.
+
+          This mode depends on a set of :ref:`ec2privileges` from the EC2 API.
+
+          Cilium can alternatively run in EKS using an overlay mode that gives
+          pods non-VPC-routable IPs.  This allows running more pods per
+          Kubernetes worker node than the ENI limit, but means that pod
+          connectivity to resources outside the cluster (e.g., VMs in the VPC
+          or AWS managed services) is masqueraded (i.e., SNAT) by Cilium to use
+          the VPC IP address of the Kubernetes worker node.  Excluding the
+          lines for ``eni.enabled=true`` and ``tunnel=disabled`` from the helm
```

Comment:
Based on the tests in https://github.com/cilium/cilium/issues/15190, in order to enable tunneling mode on EKS I think we should exclude also `ipam.mode=eni` from the list of helm flags.

I don't have enough context on `nodeinit.enabled=true` but it appears it's not required as well for tunneling mode

Reply:
``nodeinit.enabled=true`` is needed for now until we port the new init containers model from the CLI over to Helm.